README.md

```diff
@@ -87,13 +85,15 @@ flowchart LR;
 In order to use this workflow, you must already have completed the following steps:
 
 1. Created a Synapse project shared with challenge participants.
+1. Grant the DPE team `Administrator` privileges to said project.
 1. Created an evaluation queue within the Synapse project.
 1. One or more Docker images have already been submitted to your evaluation queue.
 1. Created a submission view that includes the `id` and `status` columns.
 1. Added the input data for evaluating submissions to a folder within your Synapse project.
 1. Added the groundtruth file to your Synapse project.
```

Comment:
If there are local sharing settings on the data + GT (which there probably are), the DPE team (or at least orca-service) will need to have download permissions as well.